.github/workflows/node-ci.yml

```diff
@@ -39,11 +39,11 @@ jobs:
       id: changed
 
     - id: js-files
-      name: Find .js changed files
+      name: Find .js, .json changed files
       uses: Khan/actions@filter-files-v0
       with:
         changed-files: ${{ steps.changed.outputs.files }}
-        extensions: '.js,.jsx'
+        extensions: '.js,.jsx,.json'
```

Comment:
This change will run flow and eslint when only package.json files are changed.

Reply:
I think we don't want that - we'd rather have it run when yarn.lock files run (otherwise everytime we update a script inside the package.json file it'll re-run all tests, which is super annoying)

Reply:
Good call.  I'll change it.